src/pages/instructor/dashboard/index.tsx

```diff
@@ -0,0 +1,203 @@
+                    'bg-orange-700':
+                      resource.productionProcessState === 'drafting' ||
+                      resource.productionProcessState === 'content review',
+                    'bg-green-700':
+                      resource.productionProcessState === 'pre release',
+                  },
+                )}`}
+              >
+                {resource.productionProcessState}
+              </p>
+            )}
+            <Textfit
+              mode="multi"
+              className="font-medium leading-tight flex items-center h-fit"
+              max={22}
+            >
+              <h3>{resource.title}</h3>
+            </Textfit>
+            {resource.description && describe && (
+              <ReactMarkdown className="prose dark:prose-dark prose-sm dark:text-gray-300 text-gray-700 dark:prose-a:text-blue-300 prose-a:text-blue-500 sm:block hidden">
+                {truncate(resource.description, {length: 120})}
+              </ReactMarkdown>
+            )}
+            <CardFooter>
+              <CardAuthor className="flex items-center md:pt-0 pt-2" />
+            </CardFooter>
+          </CardBody>
+        </CardContent>
+        {children}
+      </Card>
+    </ExternalTrackedLink>
+  )
+}
+
+const Dashboard: React.FunctionComponent<any> = ({courses}) => {
+  console.log('queryResult:', courses)
+  return (
+    <section className="py-10">
+      <div className="container mx-auto max-w-xl">
+        <h1 className="text-2xl mb-12">Instructor Dashboard</h1>
+        <section className="">
+          <div className="my-4">
+            <h2 className="text-xl">WIP Courses</h2>
+            <p className="m-2 text-small leading-snug">
+              This is list of your work in progress Courses. Currently when you
+              select a course, you will be redirected to app.egghead while we
+              build out the authoring experience on the new site.
+            </p>
+          </div>
+          {courses.map((course: any) => {
+            return <WIPCard resource={course} />
+          })}
+        </section>
+      </div>
+    </section>
+  )
+}
+
+const transformNameToSanityCollaborator = (name: string) => {
+  return `collaborator-instructor-${name.split(' ').join('-').toLowerCase()}`
+}
+
+export const getServerSideProps: GetServerSideProps = async function ({req}) {
+  const {eggheadToken} = getTokenFromCookieHeaders(req.headers.cookie as string)
+  const user = await fetchEggheadUser(eggheadToken, true)
+
+  const sanityCollaborator = transformNameToSanityCollaborator(user.full_name)
+  console.log({sanityCollaborator})
+
+  const ability = await getAbilityFromToken(req.cookies[ACCESS_TOKEN_KEY])
+
+  if (ability.can('upload', 'Video')) {
+    const testQuery = groq`
+    *[_type == 'course']
+     ['${sanityCollaborator}' in collaborators[]._ref] 
+     [productionProcessState != 'published' && productionProcessState != 'retired'] {
```

Comment:
filters for courses that have a collaborator of `:sanity-id` that are not in the states of `published` or `retired`